backend/capellacollab/projects/toolmodels/backups/runs/interface.py

```diff
@@ -93,35 +93,28 @@ def _schedule_pending_jobs():
             db.commit()
 
 
-def _transform_utc_to_local_timestamp(
-    timestamp: datetime.datetime,
-) -> datetime.datetime:
-    return timestamp.replace(tzinfo=pytz.UTC).astimezone()
-
-
 def _transform_kubernetes_logline_to_loki_entry(
     line: str,
 ) -> loki.LogEntry:
     datetime_without_nanoseconds = line.split()[0][:-11]
     timestamp = datetime.datetime.strptime(
         datetime_without_nanoseconds, "%Y-%m-%dT%H:%M:%S"
-    )
+    ).replace(tzinfo=datetime.UTC)
 
-    # Transform UTC from kubernetes to local timezone
-    timestamp = _transform_utc_to_local_timestamp(timestamp)
     return loki.LogEntry(timestamp=timestamp, line=line[31:])
 
 
 def _transform_kubernetes_event_to_loki_entry(
     event: k8s_client.CoreV1Event,
 ) -> loki.LogEntry:
     # Transform UTC from kubernetes to local timezone
+    event_timestamp: datetime.datetime = (
+        event.last_timestamp or event.event_time
+    )
     timestamp = (
-        _transform_utc_to_local_timestamp(
-            event.last_timestamp or event.event_time
-        )
-        if event.last_timestamp or event.event_time
-        else datetime.datetime.now()
+        event_timestamp.replace(tzinfo=pytz.UTC)
```

Comment:
I think this would reinterpret the timestamp as UTC instead of converting it – is this right? In other words, we get a UTC timestamp without attached timezone info? If we get localtime, we need to use `astimezone()` instead of `replace()`.

Reply:
The `event_timestamp` comes from the Kubernetes API as response. I did just expect that Kubernetes always runs in UTC, but this doesn't to seem the case (it's the case for all of our environment though).

I've checked the response from Kubernetes Python API and the datetime attributes are timezone-aware. 

The datetime comparision also is timezone-aware, so we don't need to replace/convert here at all. 

```py
>>> now = datetime.datetime.now(datetime.UTC)
>>> now.astimezone(datetime.UTC) - now.astimezone(zoneinfo.ZoneInfo("America/Los_Angeles"))
datetime.timedelta(0)
```

Also, `timestamp()` is timezone-aware: 

```py
>>> now.astimezone(datetime.UTC).timestamp()
1691586121.067041
>>> now.astimezone(zoneinfo.ZoneInfo("America/Los_Angeles")).timestamp()
1691586121.067041
```

I'll remove the `.replace()` here.